products/rhel/download.adoc

```diff
@@ -5,7 +5,7 @@
 
 To download Red Hat Enterprise Linux Developer Suite, which includes Red Hat Enterprise Linux 7 server, a collection of development tools, and link:#{site.base_url}/products/rhel/overview/[much more], you must have an account and need to accept the terms and conditions of the Red Hat Developer Program which provides $0 subscriptions for development use only. link:#{site.base_url}/faq[Read more] about the Red Hat Developers Program.
 
-==== link:https://developers.stage.redhat.com/download-manager/file/rhel-server-7.2-x86_64-dvd.iso[Download Red Hat Enterprise Linux 7]
+==== link:#{site.base_url}/download-manager/file/rhel-server-7.2-x86_64-dvd.iso[Download Red Hat Enterprise Linux 7]
```

Comment:
You need to use 'site.download_manager_base_url' not 'site.base_url'. Currently the above renders as: http://developers-pr.stage.redhat.com/pr/941/build/609/download-manager/file/rhel-server-7.2-x86_64-dvd.iso